File: 3.swift
import Foundation

let amount = Int(readLine()!)! // ввод количества чисел
for _ in 0...amount - 1 {
    var number = Int(readLine()!)! // ввод чисел
    var count: Int = 0
    while number != 0 {
        if (number % 10) % 2 == 0 { // проверка на чётность каждой цифры
            count += 1 // подсчёт количества
        }
        number /= 10
    }
    print("Result: ", count)
}

File: 3.cs
using System;
class Program {
    static void Main() {
        int amount = int.Parse(Console.ReadLine()); // ввод количества чисел
        for (int i = 0; i < amount; i ++) {
            int number = int.Parse(Console.ReadLine()); // ввод чисел
            int count = 0;
            while (number != 0) {
                if ((number % 10) % 2 == 0) { // проверка на чётность каждой цифры
                    count++; // подсчёт количества
                }
                number /= 10;
            }
            Console.Write("Result: " + count + "\n");
        }
    }
}
